Sources/Control/Extensions/MPVolumeView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  MPVolumeView.swift
+//  ControlKit
+//
+
+import MediaPlayer
+
+extension MPVolumeView {
+    
```

Comment:
What is your opinion on having `Logger` instead of prints? 🐼

Based on current structure, I assume, you can define different Loggers for different areas. 

```suggestion
    private static let log = Logger(subsystem: "com.ControlKit.Control", category: "MPVolumeView_Extensions")
    
```

Reply:
Great suggestion, I'll add `OSLog` 🙂‍↕️